transcom/mymove
# B 19772 change closeout queue filter be int

## [B-19772](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A946112)

## Summary

This story filters out all shipments that are not in the "Needs closeout" or "Waiting on customer" status for the SC queue

### How to test

1. Login as a service counselor
2. Complete one of the moves counseling
3. Confirm that move does not show in the queue after completion

### Backend


## Screenshots
**_CLOSE1 in "Needs Closeout" status_**
<img width="1614" alt="Screenshot 2024-07-05 at 9 06 45 AM" src="https://github.com/transcom/mymove/assets/146969726/de50fc80-c48a-482f-a687-d8067bfac22d">

**_after closing out CLOSE1 no longer in queue_**
<img width="1626" alt="Screenshot 2024-07-05 at 9 08 17 AM" src="https://github.com/transcom/mymove/assets/146969726/2bfa2f5c-b6a7-4839-a593-21762b247373">
